### DISCUSSION TITLE
Improvements to the documentation would be much appreciated

### DISCUSSION THREAD

**Discussion by NicholasJudge** [alan-turing-institute/sktime#4263] [General]
Guys, thanks a lot for creating this package, it is a profound amount of work, aids many topics that sorely need to be addressed, and I really appreciate it.  It could have a meaningful impact on the pace of scientific inquiry.  

But real talk here:  Drop whatever you are doing and get the documenation to a working place - or rename your package.  The value of sklearn is that anyone who has been working in python for a while can pick up new extensions and get down to real work.  If you are building off of that framework, you fundamentally undermine your mission - and that of people trying to get work done in a timely manner - by not starting things off with a simple, 'if you are familiar with sklearn, here are the additional steps you need to take to use sktime'.  

I click on 'user guide' and it takes me to a new menu with another option for user guide, which is then not a user guide but the six topics that are listed on previous pages.  What looks like articles are actually videos.  It's truly Kafkaesque.  There are no example calls in function documentation.  It is not clear from where to import certain functions.  I have never, in my 15 years of doing this seen such a juxtaposition between promising tech & poor documentation.   

My point is you are using your branding to suggest to developers that they can hop right in and get to work.  So you are actually causing harm by tricking people into trying things out.  

I would suggest removing all the verbiage about time series basics and just speak to an audience that has a normal dataframe with a datetime index, a corresponding dependent variable, and tell them how they can run a large suite of models right away, from whence to import them, and what they need to do to their data to get things to run.

**Comment by fkiraly** [comment]
@NicholasJudge, thanks for your encouragement to work on documentation. We know this is a point that needs improvement. See https://github.com/sktime/sktime/issues/3686.
Some points:

* I would like ask you to be kind and helpful - `sktime` is a volunteer driven, open project. Please *do refrain from putdowns* and *implications of malice* like in your message, e.g., "drop everything", "causing harm", "tricking people". I hope instead you may like to apologize and say some warm word of thanks to [all our contributors](https://github.com/sktime/sktime/blob/main/.all-contributorsrc).
* help with the documentation! This is an open project and you can jump right in.
* You could also give concrete pointers what you would suggest improving. Planning out and splitting it into small tasks is also helpful. Where have you felt documentation can be improved?
* You can also [donate money](https://github.com/sponsors/sktime) and we can pay someone to improve our documentation.

**Reply by fkiraly** [reply]
PS: have you seen our tutorials on binder?
https://mybinder.org/v2/gh/sktime/sktime/main?filepath=examples

**Comment by fkiraly** [comment]
@NicholasJudge, I actually start doubting that you are looking at the most recent version of our documentation. Google search sometimes still points you to old versions.

Are you looking at this?
https://www.sktime.net/en/stable/

>  There are no example calls in function documentation

Here's one example of example calls, and there are many more.
Admittedly, some estimators are lacking examples, so it would be great if someone added them.

![image](https://user-images.githubusercontent.com/7985502/220860553-ba1ae286-67f3-424b-8c69-e3178c103528.png)

> What looks like articles are actually videos

I think you are referring to this page: https://www.sktime.net/en/stable/user_guide.html

The links get you to notebooks, not videos?

**Reply by fkiraly** [reply]
I opened an issue to add missing examples - if you want to contribute, would be much appreciated:

https://github.com/sktime/sktime/issues/4264

**Comment by fkiraly** [comment]
>It is not clear from where to import certain functions.

This is a valid point, we weren't able to find out how you put the import statement at the top of the API ref (like sklearn) does. If you have pointers how to do this, that would be much appreciated. Or, a PR is appreciated even more!
